src/libratbag.h

```diff
@@ -1125,6 +1141,12 @@ enum ratbag_button_type {
 	RATBAG_BUTTON_TYPE_PROFILE_DOWN,
 };
 
+struct ratbag_color {
+	unsigned short red;
+	unsigned short green;
+	unsigned short blue;
+};
```

Comment:
if that is an open struct (which I'm not opposed to) you can just define it in the declaration in the hunk above.

I wonder: why did you pick unsigned short? unsigned int seems the more natural choice here (or even uint32_t if we start switching to stdint for the API, which we should)